lib/iris/tests/unit/analysis/regrid/test_RectilinearRegridder.py

```diff
@@ -1262,9 +1263,7 @@ def test_circular_src__masked_missingmask(self):
         # instead of being an array.
         src = self.src
         src.coord('longitude').circular = True
-        src_fill_value = src.fill_value
-        src.data = np.ma.MaskedArray(src.data)
-        src.fill_value = src_fill_value
+        src.replace(ma.MaskedArray(src.data), fill_value=src.fill_value)
```

Comment:
Folded in the `np.ma` -> `ma` change ... but the above use of `replace` is the primary reason for touching `test_RectilinearRegridder.py`